### ISSUE TITLE
Reconciliation API returning indexes as scores?

### ISSUE THREAD

**Issue by ianengelbrecht** [rogerhyam/wfo-plant-list#8]
Sorry to keeping hammering on here...

Using the recon service from OpenRefine, if only one candidate is returned we get a recon score for that candidate which looks like a percentage, but if more than one candidate is returned the score looks like it's the index of the candidate in the result (1, 2, 3, etc).

**Comment by rogerhyam** [comment]
Yes these scores are somewhat arbitrary. The standard doesn't define a measure and when you are doing complex things like name matching it is tough to come up with something meaningful. What counts more or less highly towards the score? 

https://www.w3.org/community/reports/reconciliation/CG-FINAL-specs-0.2-20230410/#a-note-on-candidate-retrieval-and-scoring

I just give the reverse ranking number if there are multiple candidate and 100 (top score!) if it is the only one.

I'm open to suggestions of what the scores should be based on. The main use case is simply to present the list to the user so anything other than alphabetical will probably get kickback!

**Comment by ianengelbrecht** [comment]
Okay so I've been using this form of Levenshtein to compare results from WFO with the names I have and it's relatively useful - filtering out matches below a certain threshold. It doesn't work for autonyms as autonyms don't have authors in WFO, but hopefully we'll have those at some point in future.
